README.md

```diff
@@ -1,6 +1,6 @@
 # Rusty V8 Binding
 
-V8 Version: 8.6.337, 2020-08-05
+V8 Version: 8.6.334, 2020-08-08
```

Comment:
Let's remove this date. It's unclear what it means. I always put the date of the v8 release there vs whatever today's date is.

Reply:
fine - can we do that next time?